docs/Configuring-Orion/TLS.md

```diff
@@ -3,15 +3,162 @@ description: TLS
 
 # TLS
 
-Orion supports the TLS (Transport Layer Security) protocol to enable secure communications.
+Orion supports the Transport Layer Security (TLS) protocol to enable secure communications. TLS is used between Orion nodes only. It is not used between Pantheon and Orion nodes.
 
 Enable TLS by setting the `tls` property to `strict` in the [Orion configuration file](Configuration-File.md).  
 
 ## Getting Started with TLS
 
 To have Orion generate certificates, start the nodes with the TLS trust mode set to `insecure-no-validation` for `tlsclienttrust` 
 and `tlsservertrust` until the `tlsknownclients` and `tlsknownservers` files are populated. When the files are populated, 
-restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`. 
+restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`.
+
+The following procedures explain how to generate certificates when the Common Name (CN) is either the public DNS or an IP address by using [OpenSSL](https://www.openssl.org/source/). Before you begin, ensure OpenSSL is installed.
+
+Choose one of the following methods:
+
+* [Generating certificates when the public DNS is the CN](#Generating-certificates-when-the-public-DNS-is-the-CN)
+* [Generating certificates when the public IP address is the CN](#Generating-certificates-when-the-public-IP-address-is-the-CN)
+
+### Generating certificates when the public DNS is the CN
+
+If you want to use a public DNS as the CN, complete the following steps:
+
+#### Generating a CA certificate
+
+1. Generate a key file called orion_ca.key by running the following command:
+
+    `openssl genrsa -out orion_ca.key 2048`
+  
+1. Generate a CA certificate called orion_ca.pem that uses orion_ca.key by running the follow command:
+
+    `openssl req -x509 -new -nodes -key orion_ca.key -sha256 -days 1024 -out orion_ca.pem`
+
+#### Generating a new certificate for a node
```

Comment:
I'm confused -> what's the difference between a _CA certificate_ and this new cetificate?  Is this obvious to users who understand TLS (ie, it's just me that's confused).  @lucassaldanha

Reply:
The CA (certificate authority) is the certificate that will be used to "sign" the other certificates. Usually, the way it works is that you pay a trusted certificate authority company to give you a certificate signed by them.

In the case that you want to be your own certificate authority, you need to generate the CA first, and then the certificates used by the nodes.